app/bundles/CoreBundle/Assets/json/countries.json

```diff
@@ -82,8 +100,10 @@
     "Ireland",
     "Israel",
     "Italy",
+    "Ivory Coast",
     "Jamaica",
     "Japan",
+    "Jersey",
     "Jordan",
     "Kazakhstan",
```

Comment:
We seem to be missing both Koreas ("Democratic People's Repulic of" AKA "North", and "Republic of" AKA "South")